forge-gui/res/cardsfolder/upcoming/minthara_of_the_absolute.txt

```diff
@@ -0,0 +1,12 @@
+Name:Minthara of the Absolute
+ManaCost:2 W B
+Types:Legendary Creature Elf Cleric
+PT:2/4
+K:Starting intensity:0
+T:Mode$ ChangesZone | ValidCards$ Permanent.YouCtrl | Origin$ Battlefield | Destination$ Any | TriggerZones$ Battlefield | Execute$ TrigIntensify | ActivationLimit$ 1 | TriggerDescription$ Whenever one or more permanents you control leave the battlefield, perpetually increase NICKNAME's intensity and the intensity of cards named Minthara of the Absolute in your hand, graveyard, and library by 1. This ability triggers only once each turn.
+SVar:TrigIntensify:DB$ Intensify | AllDefined$ Card.inZoneBattlefield+namedMinthara of the Absolute+YouOwn,Card.inZoneGraveyard+namedMinthara of the Absolute+YouOwn,Card.inZoneHand+namedMinthara of the Absolute+YouOwn,Card.inZoneLibrary+namedMinthara of the Absolute+YouOwn | DefinedDesc$ Perpetually increase the intensity of Minthara of the Absolute and the intensity of cards named Minthara of the Absolute in your hand, graveyard, and library by 1.

```

Comment:
`AllDefined$ Card.Self,Card.inZoneGraveyard+namedMinthara of the Absolute+YouOwn,Card.inZoneHand+namedMinthara of the Absolute+YouOwn,Card.inZoneLibrary+namedMinthara of the Absolute+YouOwn`
- the card doesn't say anything about other cards on the battlefield (thinking of something like Spark Double)

all that `DefinedDesc$` can be removed... it only matters for StackDescription, which is handled by line 6 anyway

Reply:
Good catch !